migrations/versions/5ca7df80b59f_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: 5ca7df80b59f
+Revises: 011c3345849a
+Create Date: 2022-05-04 08:53:42.074660
+
+"""
+from alembic import op
```

Comment:
Just curious about how this migration comes in with the K8s version update. Please enlighten me @Mubangizi. Also is it possible to update all staging clusters as Inno suggested for easy testing?